appveyor.yml

```diff
@@ -0,0 +1,27 @@
+# Set up the build environment.
+environment:
+    matrix:
+        - TARGET: x86_64-pc-windows-msvc
+        - TARGET: i686-pc-windows-msvc
+        - TARGET: i686-pc-windows-gnu
+
+# Set up the Rust toolchain.
+install:
+    - ps: Start-FileDownload "https://static.rust-lang.org/dist/rust-nightly-${env:TARGET}.exe"
```

Comment:
It looks like appveyor build fails, because it uses nightly. Changing toolchain to stable should fix it.